chart/epinio/values.yaml

```diff
@@ -5,9 +5,19 @@
 # The email address you are planning to use for getting notifications about your certificates.
 email: "epinio@suse.com"
 
+# The URL of the container registry from where to pull container images
+# for the various created Pods. Don't confuse this registry with the "Epinio
+# registry" which is the one where Epinio stores the application images.
+registryURL: ""
+
 # The system domain that will be used to access the epinio API server
 systemDomain: ""
 
+tekton:
```

Comment:
What do you think to use the following format:
```
image:
    epinio:
        registry: splatform/epinio-server
        tag: v0.2.1
    bash:
        registry: library/bash (or just bash)
        tag: 5.1.4
    aws-cli:
        registry: amazon/aws-cli
        tag: 2.0.52
    kubectl: 
        registry: lachlanevenson/k8s-kubectl
        tag: v1.22.2
```

It's a personal taste but I prefer to have a different field for the tag as it's going to change more often

Reply:
Looks cleaner, yes. I'm not sure we should call the key `registry` though. It will either be the combination of a registry + an image (e.g. `ghcr.io/epinio/epinio-server`) or just the image (e.g. `splatform/epinio-server`). So the part that's always there is the image.

Maybe we should remove the top-level `registryURL` and let users specify each image repository separately?

Reply:
I am sorry, I meant "repository" and not registry. I don't think being able to override the default registry in some situations would be useful at least at this stage but I already see the value of being able to override the default registry URL to let say a local proxy cache.

Reply:
```
image:
    epinio:
        repository: splatform/epinio-server
        tag: v0.2.1
    bash:
        repository: library/bash (or just bash)
        tag: 5.1.4
    aws-cli:
        repository: amazon/aws-cli
        tag: 2.0.52
    kubectl: 
        repository: lachlanevenson/k8s-kubectl
        tag: v1.22.2
```

Reply:
oh it makes more sense now :) . I agree. Let me change things.